pep-0011.txt

```diff
@@ -158,13 +158,15 @@ Microsoft has established a policy called product support lifecycle
 [1]_. Each product's lifecycle has a mainstream support phase, where
 the product is generally commercially available, and an extended
 support phase, where paid support is still available, and certain bug
-fixes are released (in particular security fixes).
+fixes are released. Sometimes this is then followed by a third phase
+called "ESU", for "extended security updates".
```

Comment:
Perhaps mention ESU is a program of last resort? And link to https://learn.microsoft.com/en-us/lifecycle/faq/extended-security-updates